lms/static/js/i18n/ru/djangojs.js

```diff
@@ -1374,7 +1374,7 @@
     "Showing {currentItemRange} out of {totalItemsCount}, sorted by {sortName} descending": "\u041e\u0442\u043e\u0431\u0440\u0430\u0436\u0430\u0435\u0442 {currentItemRange} \u0438\u0437 {totalItemsCount},  \u0441\u043e\u0440\u0442\u0438\u0440\u043e\u0432\u043a\u0430 {sortName} \u043f\u043e \u0443\u0431\u044b\u0432\u0430\u043d\u0438\u044e.", 
     "Showing {firstIndex} out of {numItems} total": "\u041f\u043e\u043a\u0430\u0437\u044b\u0432\u0430\u0442\u044c \u0441\u043e\u0432\u043e\u043a\u0443\u043f\u043d\u043e\u0441\u0442\u044c {firstIndex} \u043a {numItems} ",
     "Showing {firstIndex}-{lastIndex} out of {numItems} total": "\u041f\u043e\u043a\u0430\u0437\u044b\u0432\u0430\u0442\u044c \u0441\u043e\u0432\u043e\u043a\u0443\u043f\u043d\u043e\u0441\u0442\u044c {firstIndex}-{lastIndex} \u043a {numItems} ", 
-    "Sign In": "\u0420\u0435\u0433\u0438\u0441\u0442\u0440\u0430\u0446\u0438\u044f", 
+    "Sign In": "\u0412\u043e\u0439\u0442\u0438", 
```

Comment:
Please provide a respective re-compiled mo file @RuslanSkiraRaccoongang

Reply:
Discussed; it turns out mo file was already changed (somehow). Worked on local devstack, let's check on stage.